app/routes.js

```diff
@@ -68,6 +73,16 @@ router.get('*/eligibility', (req, res, next) => {
   next();
 });
 
+router.get('*/already-notified', (req, res, next) => {
+  const data = req.session.data;
+
+  if (!data['deceased-national-insurance'] || data['deceased-national-insurance'] === '') {
```

Comment:
I think this first check would be the same as last, im sure `''` is falsey so wouldn't need both